### ISSUE TITLE
Contracts 500 error

### ISSUE THREAD

**Issue by gmcintire** [SonarSoftwareInc/customer_portal#5]
Enabling contracts in settings and then logging in as a user and going to settings screen results in 500 error. In this specific instance my user has 1 contract signed.

**Comment by zulandar** [comment]
@gmcintire this issue is due to the customer_portal_framework attempting to pull from .env file that doesn't exist in the new sonar customer portal. I've submitted a PR for that fix.

**Comment by zulandar** [comment]
This has been updated within the customer_portal_framework and the customer_portal PR #6 upgrades the version within the customer portal.

**Comment by zulandar** [comment]
@gmcintire this issue was resolved in the last build. Feel free to reopen if your still experiencing an issue.

**Comment by gmcintire** [comment]
This is still broken as of the latest watchtower pull:

 time="2019-05-01T18:04:03Z" level=info msg="Found new sonarsoftware/customerportal:latest image (sha256:35b78e4838a596302815a00cbaeac320e33ed26adcfa18bec23f9603e1fa5a02)"

**Comment by gmcintire** [comment]
I just deployed a brand new instance on DO and am still getting a 500 on viewing contracts.

**Comment by zulandar** [comment]
@gmcintire I am reopening the issue and will test in a fresh instance today. 

Can you provide the `sudo docker-compose logs app` on the fresh install after you have attempted to access the contracts?

**Comment by gmcintire** [comment]
The only logs I get from sonar-customerportal is:

```
sonar-customerportal | [Fri May 03 17:58:46.057112 2019] [log_config:warn] [pid 70] (32)Broken pipe: [client 172.18.0.6:46088] AH00646: Error writing to /var/log/apache2/access.log, referer: https://portal.vntx.net/portal/billing
```

**Comment by zulandar** [comment]
PR #7 should resolve this issue with the Contracts page.

**Comment by geneccx** [comment]
Pr #7 has been built and pushed up to the registry and should be live within the next 5 minutes.

Feel free to reopen this issue if you encounter the error again.

**Comment by gmcintire** [comment]
Watchtower pulled the latest update a few minutes ago and the contracts page is still giving a 500 error.

```
nginx_1       |  - - [13/May/2019:17:24:03 +0000] "GET /portal/contracts HTTP/1.1" 500 1458 "https://portal.vntx.net/portal/billing" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safa
```

**Comment by SonarSoftware** [comment]
@zulandar Let's get this setup in an identical Docker container and tested so we can get this solved.

**Comment by zulandar** [comment]
@SonarSoftware @gmcintire - Issue with composer.lock file being out of date. Fixed in commit d1d4b58045a585f6fc5e66db29afa74bdbf89a5d and tested on identical docker container setup.

**Comment by gmcintire** [comment]
Can confirm it works now, thanks!
